### PR TITLE
fix: reload at remove file

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- reload when remove file
- ~~debounce will trigger immediate~~ 这个还是不用了.

```js
    // some editor will use temp file, so had to check info.remove
    // vim change mode:
    //  - {"event":"rename","path":"/PATH_TO_DIR/home.js","stat":{...},"remove":false,"isDirectory":false,"isFile":true}
    //  - {"event":"rename", "path":"/PATH_TO_DIR/home.js~", "remove":true, "isDirectory":false, "isFile":false }
    // webstorm: 
    //  - {"event":"rename","path":"/PATH_TO_DIR/home.js___jb_old___","remove":true,"isDirectory":false,"isFile":false}
```

### CODE REVIEW

**Comment by atian25** [comment]
那两个 release 的, 好像是上次权限问题, 导致 master 没推上去导致的.

**Comment by jtyjty99999** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "egg-development",
-  "version": "0.0.2",
+  "version": "1.0.0",
```
Comment (jtyjty99999) [inline package.json]: 这些是什么情况，去掉吧，我单独commit一个release

Comment (atian25) [inline package.json]: 那两个 release 的, 好像是上次权限问题, 导致 master 没推上去导致的.

Comment (atian25) [inline package.json]: 我来吧, 我先 push 下 master

Comment (atian25) [inline package.json]: @jtyjty99999 可以了

**Comment by atian25** [comment]
1.0.1

**Pull request merged**